rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -814,6 +812,63 @@ void usb_handler_thread::push_fake_transfer(UsbTransfer* transfer)
 	fake_transfers.push_back(transfer);
 }
 
+const std::array<u8, 7>& usb_handler_thread::get_new_location()
+{
+	location[0]++;
+	return location;
+}
+
+void usb_handler_thread::connect_usb_device(std::shared_ptr<usb_device> new_dev)
+{
+	usb_devices.push_back(new_dev);
+
+	for (auto iterator = ldds.begin(); iterator != ldds.end(); iterator++)
+	{
+		sys_usbd.success("connect_usb_device : check ldd : name=%s VID=%04x, PID=%04x->%04x", iterator->first,
+			iterator->second.id_vendor, iterator->second.id_product_min, iterator->second.id_product_max);
+		if (new_dev->device._device.idVendor == iterator->second.id_vendor && new_dev->device._device.idProduct >= iterator->second.id_product_min && new_dev->device._device.idProduct <= iterator->second.id_product_max)
+		{
+			int assigned_number = dev_counter++;
+			handled_devices.emplace(assigned_number, std::pair(UsbInternalDevice{narrow<u8>(assigned_number), narrow<u8>(assigned_number), 0x00, 0x00}, new_dev));
+			new_dev->assigned_number = assigned_number;
+			send_message(SYS_USBD_ATTACH, assigned_number);
+			sys_usbd.success("connect_usb_device : ldds already registered : assigned_number = %d", assigned_number);
+		}
+	}
+}
+
+void usb_handler_thread::disconnect_usb_device(std::shared_ptr<usb_device> dev)
+{
+	if (dev->assigned_number && handled_devices.erase(dev->assigned_number))
+	{
+		sys_usbd.success("disconnect_usb_device : detach device %d", dev->assigned_number);
+		send_message(SYS_USBD_DETACH, dev->assigned_number);
+		dev->assigned_number = 0;
+	}
+
+	usb_devices.erase(find(usb_devices.begin(), usb_devices.end(), dev));
+}
+
+void connect_usb_controller(u8 index, input::product_type type)
+{
+	bool already_connected = false;
+	auto& usbh = g_fxo->get<named_thread<usb_handler_thread>>();
+
+	if (const auto iterator = usbh.pad_to_usb.find(index); iterator != usbh.pad_to_usb.end())
```

Comment:
It's called 'it' everywhere else, since iterator is the type...
So why not do it like everyone else?

Reply:
In sys_usbd.cpp on master it is used two times "iterator" and only once "it"